bleak/backends/bluezdbus/utils.py

```diff
@@ -43,3 +45,17 @@ def bdaddr_from_device_path(device_path: str) -> str:
         A Bluetooth address as a string.
     """
     return ":".join(device_path[-17:].split("_"))
+
+
+def get_dbus_authenticator():
+    uid = 0
+    try:
+        uid = int(os.environ("BLEAK_DBUS_UID"))
+    except ValueError:
+        pass
+
+    auth = None
+    if uid:
+        auth = AuthExternal(uid=uid)
+
+    return auth
```

Comment:
This would not allow `BLEAK_DBUS_UID=0`. We can allow that by using `None` instead of `0`.

```suggestion
def get_dbus_authenticator():
    uid = None
    try:
        uid = int(os.environ("BLEAK_DBUS_UID"))
    except ValueError:
        pass

    auth = None
    if uid is not None:
        auth = AuthExternal(uid=uid)

    return auth
```

Reply:
Good catch.